### ISSUE TITLE
[Bug] Crash with WilderWild and Moonlight Lib

### ISSUE THREAD

**Issue by SplendidAlakey** [quiqueck/BetterNether#48]
### What happened?

Crash on startup.

To reproduce:

1. Install WilderWild, Moonlight and BCLib
2. Install either BetterEnd or BetterNether (or both, in which case BetterNether will be blamed for the crash)
3. Launch the game and crash

BetterX mods don't seem to be incompatible with WilderWild on their own, but only with Moonlight present.

Removing either WilderWild, Moonlight Lib or both BetterX mods lets the game start normally. Initially reported to WilderWild here: https://github.com/Soro335/WilderWild/issues/85. Since then every mod on the list has been updated, but the crash is still reproducible. I updated the logs to reflect that. Also reported to Moonlight Lib: https://github.com/MehVahdJukaar/Moonlight/issues/31

### BetterNether

7.1.0

### BCLib

2.1.0

### Fabric API

0.60.0

### Fabric Loader

Quilt 0.17.3

### Minecraft

1.19.1

### Relevant log output

```shell
Logs include latest.log and crash log for the BetterEnd crash, but only the latest.log for BetterNether, because it does not generate a crash log: https://gist.github.com/Footage2-Amply-Pounce/e7506db63847f6f7f315af3cddee93c6
```

### Other Mods

```shell
**[QSL](https://modrinth.com/mod/qsl)** 4.0.0 beta 9
**[WilderWild](https://modrinth.com/mod/wilder-wild)** 1.0.5
**[Moonlight Lib](https://www.curseforge.com/minecraft/mc-mods/selene)** 2.0.17 (it's not needed for any of the mods, only to reproduce the crash; I personally need it for Supplementaries which doesn't have to be installed to reproduce the crash)
**BCLib** 2.1.0
**BetterEnd** 2.1.0

Minecraft version is 1.19.2, but it's not an option in the dropdown menu.
```

**Comment by Surpos0** [comment]
i have same issue but on 1.19

**Comment by Xfrtrex** [comment]
WilderWild dev here: please fix, seems an issue with BetterX as it crashes without moonlight too.

**Comment by quiqueck** [comment]
As far as I can see, something in Moonlight Lib causes our features to get loaded before Vanillas `NoneFeatureConfiguration.NONE` was initialized. I am not sure what code is causing this tho

**Comment by quiqueck** [comment]
Should be fixed in BCLib 2.1.1. Please re-open this issue if the error is still there
